### PR TITLE
Np 865 using layers for publication commons

### PR DESCRIPTION
**Pull request by janbj**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,6 @@
 plugins {
     id 'java'
+    id 'java-library'
```
Comment (axthosarouris) [inline build.gradle:3]: do you need both 'java' and 'java-library'?

File: publication-commons/build.gradle [hunk 2]
```diff
@@ -2,7 +2,22 @@ repositories {
     maven { url "https://s3.eu-central-1.amazonaws.com/dynamodb-local-frankfurt/release" }
 }
 
+def jacksonVersion = '2.10.2'
+def awsSdkVersion = '1.11.714'
+
 dependencies {
+    api group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: 'v0.3.0'
```
Comment (axthosarouris) [inline publication-commons/build.gradle:9]: Are you sure that implementation does not do the job?

Comment (janbj) [reply publication-commons/build.gradle:9]: Yes, I didn't get access to transitive dependencies when using implementation

File: build.gradle (hunk 1 above)
Comment (janbj) [reply build.gradle:3]: Maybe not?

**Pull request merged**